src/components/Footer/Footer.js

```diff
@@ -19,9 +19,10 @@ function Footer() {
             }}>
                 <ul>
                     <li><h1>Community</h1></li>
+                    <li><a href='https://discord.gg/invite/TSU6StNQUG'>CCCP Development Discord</a></li>
```

Comment:
Thoughts about moving this under the "development" section?

Reply:
That seems reasonable to me, will do.